doc/_extensions/zephyr/gh_utils.py

```diff
@@ -46,12 +47,33 @@
 from sphinx.application import Sphinx
 from sphinx.util.i18n import format_date
 
+ZEPHYR_BASE = Path(__file__).parents[3]
+SCRIPTS = ZEPHYR_BASE / "scripts"
```

Comment:
I see that we are using Python typehints below for some functions. We can expand that to things like this too:
```suggestion
ZEPHYR_BASE : Final[str] = Path(__file__).parents[3]
SCRIPTS : Final[str] = ZEPHYR_BASE / "scripts"
```

(requires `from typing import Final`)

Reply:
+1, will do

Reply:
done -- also made MAINTAINERS final.